saleor/graphql/shop/mutations.py

```diff
@@ -288,3 +288,92 @@ def perform_mutation(cls, _root, info, key_type):
 
         instance.delete()
         return AuthorizationKeyDelete(authorization_key=instance, shop=Shop())
+
+
+class StaffNotificationRecipientInput(graphene.InputObjectType):
+    user = graphene.ID(required=False, description="The ID of the user to notify.")
+    staff_email = graphene.String(
+        required=False, description="Email to notify instead of given staff member."
```

Comment:
Please use more or less the same descriptions that we have in our `StaffNotificationRecipient` type (I made a few suggestions there).